docs/source/plugins.rst

```diff
@@ -216,6 +216,13 @@ plugin:
        ``dataset`` (an instance of ``mlflow.models.evaluation.base._EvaluationDataset`` containing features and labels (optional) for model evaluation),
        ``run_id`` (the ID of the MLflow Run to which to log results), and ``evaluator_config`` (a dictionary of additional configurations for the evaluator).
      - `DummyEvaluator <https://github.com/mlflow/mlflow/blob/branch-1.23/tests/resources/mlflow-test-plugin/mlflow_test_plugin/dummy_evaluator.py>`_.
+   * - Plugins for custom mlflow server flask app configuration `mlflow.server.app <https://github.com/mlflow/mlflow/blob/v2.1.1/mlflow/server/__init__.py#L30>`_.
```

Comment:
Unsure which tag to use here since it won't show until released. Can adjust based on release schedule or change to this commit.

Reply:
Let's use v2.0.0!